snapshot/snapshot.go

```diff
@@ -268,37 +281,54 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 			return nil, err
 		}
 	}
+
 	if target, ok := base.Labels[targetSnapshotLabel]; ok {
+
 		// NOTE: If passed labels include a target of the remote snapshot, `Prepare`
 		//       must log whether this method succeeded to prepare that remote snapshot
 		//       or not, using the key `remoteSnapshotLogKey` defined in the above. This
 		//       log is used by tests in this project.
 		lCtx := log.WithLogger(ctx, log.G(ctx).WithField("key", key).WithField("parent", parent))
-		if err := o.prepareRemoteSnapshot(lCtx, key, base.Labels); err != nil {
-			log.G(lCtx).WithField(remoteSnapshotLogKey, prepareFailed).
-				WithError(err).Warn("failed to prepare remote snapshot")
-			if !errors.Is(err, ErrNoZtoc) {
-				commonmetrics.IncOperationCount(commonmetrics.FuseMountFailureCount, digest.Digest(""))
+
+		localOverride := false
+		if strVal, ok := base.Labels["containerd.io/snapshot/remote/image.layers.size"]; ok {
```

Comment:
nit: can we export this field and use that instead? 

https://github.com/awslabs/soci-snapshotter/blob/main/fs/source/source.go#L95